pkg/trace/api/version.go

```diff
@@ -0,0 +1,76 @@
+package api
+
+// Version is a dumb way to version our collector handlers
+type Version string
+
+const (
+	// v01 DEPRECATED, FIXME[1.x]
+	// Traces: JSON, slice of spans
+	// Services: deprecated
```

Comment:
Nit: Is it still relevant to refer to "services" endpoints here?
Since it got mostly strip away from the code, we may want to remove it from here to, helping any new reader.